requirements.txt

```
@@ -1,5 +1,5 @@
 sphinx==3.2.0
 sphinx-autobuild==0.7.1
-sphinx-inline-tabs==2021.3.19b5
+sphinx-inline-tabs==2021.4.11b9
```

Maybe try triple equals?
```suggestion
sphinx-inline-tabs === 2021.4.11b9
```

---

The problem is that sphinx-inline-tabs requires Python 3.8+. Can that be changed on the readthedocs config in the GUI? I don't see a `.readthedocs.yml` to change it in locally.

---

Actually, it looks like it can only be changed via a config file. The GUI choices are 2.x and 3.x.